tests/event/behavior/keydown.ts

```diff
@@ -317,6 +317,8 @@ cases(
       <input type="radio" name="" value="nameless2"/>
       <input type="radio" name="group" value="c" disabled/>
       <input type="radio" name="group" value="d"/>
+      <input type="radio" name="group" value="e"/>
+      <input type="radio" name="group" value="f"/>
```

Comment:
There were only two available radio buttons actually (the values are `"a"` and `"d"`).
I've added some other elements to make the behavior easier to understand.